docker/Dockerfile

```
@@ -0,0 +1,72 @@
+        $([ -f ./Package.resolved ] && echo "--force-resolved-versions" || true)
+
+COPY src src
+COPY --from=client-build /home/app/Resources/Views/react.leaf /home/vapor/Resources/Views/react.leaf
+ 
+RUN swift sdk list
+RUN swift build -c release
+
+FROM ubuntu:noble
+
+# Make sure all system packages are up to date, and install only essential packages.
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)